quantum/split_common/split_util.c

```diff
@@ -31,7 +31,7 @@
 #    define SPLIT_USB_TIMEOUT_POLL 10
 #endif
 
-#ifdef PROTOCOL_CHIBIOS
+#if defined(PROTOCOL_CHIBIOS) || defined(BOOTLOADER_HALFKAY)
```

Comment:
From the chat on discord, i'm really not a fan of this coupling. Without knowing the underlying context around vbus detection, all i'm left with is `why is there bootloader logic within split code?`. I don't have another suggestion right now but will give it a think.

(For reference, I also don't like the existing `PROTOCOL_CHIBIOS` logic being here either.)

Reply:
Agreed, but until/unless a better option exists...

Could also move to bootloader.mk but I feel that makes less sense

Reply:
I may have a better option, actually.  

If that works, I'll close this, and then open a PR with documentation for that solution.